slack_backup/client.py

```diff
@@ -154,12 +155,16 @@ def _create_message(self, data, channel):
 
         logging.debug('Message data: %s', json.dumps(data))
 
-        try:
-            user = self.q(o.User).\
-                filter(o.User.slackid == data['user']).one()
-        except KeyError:
-            user = self.q(o.User).\
+        if 'user' in data:
```

Comment:
With this condition, we will never end up with `KeyError` exception, so that we will never reach line 162.